article/templatetags/tests/test_article_tags.py

```diff
@@ -0,0 +1,19 @@
+from django.template.exceptions import TemplateDoesNotExist
+import pytest
+
+from article.templatetags import article_tags
+
+
+def test_include_markdown_valid_template():
```

Comment:
would be good to try to render all markdown files in this test, hopefully some exception would be raised if the syntax is invalid - if not can you add such

Reply:
we do - in the `test_article_views.py`